source/documentation/add-packages.html.md

```diff
@@ -20,35 +20,39 @@ following the best practices described in [RDO OpenStack Packaging Guidelines](/
 2. Send a review adding the new project in rdo.yml to the [rdoinfo project in
 review.rdoproject.org](https://review.rdoproject.org/r/#/q/project:rdoinfo). In
 this change you must provide the project information and Package Review bugzilla
-ticket in the commit message (see [this example](https://review.rdoproject.org/r/#/c/1408/)).
-In the project definition in **rdo.yml** file, only `under-review` must be uncommented as project tags
-and comment all releases where package should be built, as for example:
+ticket in the commit message (see [this example](https://review.rdoproject.org/r/#/c/18644/)).
+Add the project definition in **rdo.yml** file and `under-review` tag in **tags/under-review.yml** file,
+as for example:
     
-        - project: murano-dashboard
-          name: openstack-murano-ui
+        # in rdo.yml
+        - project: octavia-lib
+          conf: rpmfactory-lib
+          maintainers:
+          - nmagnezi@redhat.com
+          - cgoncalves@redhat.com
+          - bcafarel@redhat.com
+
+        # in tags/under-review.yml
+        - project: octavia-lib
           tags:
             under-review:
-            #newton-uc:
-            #newton:
-            #mitaka:
-          conf: rpmfactory-core
-          maintainers:
-          - atsamutali@mirantis.com
-          - iyozhikov@mirantis.com
      
     **Note:** Maintainers must be registered in review.rdoproject.org and use the registered email in the rdoinfo review.
     This is required to set your permissions on your project.
 
-    As part of the review process, some tasks will be carried out by the RDO team:
+    Once the patch is merged, following tasks are done by RDO automation process:-
 
-    * The required projects will be created in [https://review.rdoproject.org](https://review.rdoproject.org).
-    * Users included in maintainers list will received required permissions to manage the project.
-    * The new projects will be added to zuul configuration in softwarefactory-project.io
-    (as in [this example](https://softwarefactory-project.io/r/13535))
-    * Once the projects are created, the change will be merged in rdoinfo project.
+    * Patch is proposed to create project and assign permissions to maintainers to manage the project.
+    (as in [this example](https://review.rdoproject.org/r/#/c/18647/))
+    * Once the create project patch is merged, required projects will be created in [https://review.rdoproject.org](https://review.rdoproject.org)
+    with repo synched to [github.com](https://github.com/rdo-packages/octavia-lib-distgit).
+    * Patch is proposed to add new projects to zuul configuration in softwarefactory-project.io(now this is moved to rdoproject.org).
```

Comment:
I'd remove the reference to softwarefactory-project.io (now it's in rdoproject.org)